verticaltabs.jsm

```diff
@@ -588,17 +589,27 @@ VerticalTabs.prototype = {
     } else {
       aTab.setAttribute('crop', 'end');
     }
-
-    aTab.addEventListener('load', function () {
-      let tab_address = aTab.linkedBrowser.currentURI.spec;
-      //use to set preview image as metadata image 2/4
-      // document.getAnonymousElementByAttribute(aTab, 'anonid', 'tab-meta-image').style.backgroundImage = `url(${this.getPageMetaDataImage(aTab)}`;
-
-      //use to set preview image as screenshot
-      document.getAnonymousElementByAttribute(aTab, 'anonid', 'tab-meta-image').style.backgroundImage = `url(moz-page-thumb://thumbnail/?url=${encodeURIComponent(tab_address)}), url(resource://tabcenter/skin/blank.png)`;
-
-      document.getAnonymousElementByAttribute(aTab, 'anonid', 'address-label').value = tab_address;
-    }.bind(this));
+    let tab_meta_image = document.getAnonymousElementByAttribute(aTab, 'anonid', 'tab-meta-image');
+    tab_meta_image.background = `url(moz-page-thumb://thumbnail/?url=${encodeURIComponent(aTab.linkedBrowser.currentURI.spec)}), url(resource://tabcenter/skin/blank.png)`;
+    tab_meta_image.style.backgroundImage = tab_meta_image.background;
+
+    aTab.addEventListener('load', () => {
+      if (aTab.VTLastUrl !== aTab.linkedBrowser.currentURI.spec) {
+        if (aTab.linkedBrowser.currentURI.spec === 'about:newtab' || aTab.linkedBrowser.currentURI.spec === 'about:blank') {
+          tab_meta_image.background = 'url("resource://tabcenter/skin/newtab.png")';
```

Comment:
it gets set after the load event is finished. 
I end up writing this `tab_meta_image.style.backgroundImage = tab_meta_image.background;` 3 times, may as well just not use tab_meta_image.background.